lms/static/scripts/test-util/wait.js

```diff
@@ -61,3 +61,26 @@ export function waitForElement(wrapper, selector, timeout = 10) {
     `"${selector}" to render`
   );
 }
+
+/**
+ * Wait up to `timeout` ms for an element to be removed.
+ *
+ * @param {CommonWrapper} wrapper - Root Enzyme wrapper
+ * @param {string|Function} selector - Selector string or function to pass to `wrapper.find`
+ * @param {number} timeout
+ * @return {Promise<CommonWrapper>}
+ */
+export function waitForElementToRemove(wrapper, selector, timeout = 10) {
```

Comment:
```suggestion
export function waitForElementToBeRemoved(wrapper, selector, timeout = 10) {
```

The meaning of the previous name was obvious, this is just me being a grammar pedant.